src/Sticky.jsx

```diff
@@ -312,7 +312,7 @@ class Sticky extends Component {
         }
 
         return (
-            <div ref='outer' className={classNames('sticky-outer-wrapper', self.props.className)} style={outerStyle}>
+            <div ref='outer' className={classNames('sticky-outer-wrapper', self.props.className, {[self.props.activeClass]: self.state.status === STATUS_FIXED})} style={outerStyle}>
```

Comment:
It is my first time to see the usage 
```js
{ [someVariable]: aValue }
```
which is pretty cool. My only concern is if it is cross-browser usage? Or actually you can use simpler one like
```js
classNames('sticky-outer-wrapper', self.state.status === STATUS_FIXED ? self.props.activeClass : '')
```